compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -1,313 +1,328 @@
-// Copyright 2021 The IREE Authors
```

Comment:
what happened to the file to make it appear as diffed? did the wrong line endings or something creep in?

Reply:
Yeah, looks like vscode is going haywire. I have been having problems with it all day since an extension decided to update.

Reply:
Should be fixed now.